src/components/MarkerPopup/DesktopPopup.jsx

```
@@ -0,0 +1,25 @@
+
+    useEffect(() => {
+        if (popupRef.current) {
+            const marker = popupRef.current._source;
+            if (marker) {
+                marker.openPopup();
+            }
+        }
+    }, []);
+
+    return <StyledPopup ref={popupRef}>{children}</StyledPopup>;
+};
```

ESLint

[error] Prefer default export on a file with single export.

(import/prefer-default-export)